src/main/java/org/opensrp/repository/postgres/LocationRepositoryImpl.java

```diff
@@ -714,6 +773,10 @@ private LocationMetadata createMetadata(PhysicalLocation entity, Long id) {
 			locationMetadata.setVersion(entity.getProperties().getVersion());
 		}
 		locationMetadata.setServerVersion(entity.getServerVersion());
+
+		if(id != null){
```

Comment:
Do we need to add the complementary branch of this if condition for created at?

Reply:
https://github.com/OpenSRP/opensrp-server-core/blob/acbdf68075023cfb16a524f1ace88a6218c69b42/src/main/java/org/opensrp/repository/postgres/LocationRepositoryImpl.java#L171